imap_processing/cdf/tests/test_cdf_attribute_manager.py

```diff
@@ -164,9 +197,6 @@ def test_global_attribute():
         if required_schema is True:
             assert attr_name in test_get_global_attrs.keys()
 
-    # Testing second elif statement
-    # Should throw error
```

Comment:
If this is testing that the code successfully throws an error, you can use `pytest.raises`. If this test is failing because the code it's testing is incomplete or wrong, but it shouldn't, that's when you use `pytest.mark.xfail`. Basically, if someone should come back and fix it, `xfail` is your friend.